easybuild/easyconfigs/f/FastaIndex/FastaIndex-0.11rc7-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'FastaIndex'
+version = '0.11rc7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/lpryszcz/FastaIndex'
+description = "FastA index (.fai) handler compatible with samtools faidx"
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('Python', '2.7.14')]
+
+sanity_check_paths = {
+    'files': ['bin/FastaIndex', 'bin/fasta_stats'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+options = {'modulename': name}
```

Comment:
@boegel - isn't this the default anyway?

Reply:
The default is to use `name.lower()`, since lowercase Python package names are way more common, these are exceptions.